charts/templates/statefulset.yaml

```diff
@@ -8,6 +8,8 @@ metadata:
 spec:
   serviceName: {{ include "marklogic.headlessServiceName" . }}
   replicas: {{ .Values.replicaCount }}
+  updateStrategy:
```

Comment:
Why is this changed in this PR? This change is done under a separate issue.

Reply:
That's because I've updated the statefulset I had locally with the one from the develop Branch.
I'll fix it.